src/gluonts/mx/block/quantile_output.py

```diff
@@ -96,21 +143,19 @@ def hybrid_forward(
             y_pred_all = [F.squeeze(y_pred, axis=-1)]
 
         qt_loss = []
-        for i, y_pred_q in enumerate(y_pred_all):
-            q = self.quantiles[i]
-            weighted_qt = (
-                self.compute_quantile_loss(F, y_true, y_pred_q, q)
-                * self.quantile_weights[i]
+        for level, weight, y_pred_q in zip(
+            self.quantiles, self.quantile_weights, y_pred_all
+        ):
+            qt_loss.append(
+                weight * self.compute_quantile_loss(F, y_true, y_pred_q, level)
             )
-            qt_loss.append(weighted_qt)
         stacked_qt_losses = F.stack(*qt_loss, axis=-1)
         sum_qt_loss = F.mean(
             stacked_qt_losses, axis=-1
         )  # avg across quantiles
```

Comment:
@dcmaddix is `F.mean` correct here? Shouldn't this be `F.sum`? Since we are already weighting the quantile loss terms

Reply:
Yes, I'm not sure if we need `F.mean` since you're right we take the weighted sum on line 150. Best to confirm with @youngsuk0723.